server/routes/logins.js

```diff
@@ -13,7 +13,28 @@ router.post('/register', (req, res) => {
   hashedPassword.then((hashed) => {
    return User.create({first_name: firstName, last_name: lastName, email: email, phone_number: phone, password: hashed})
   }).then((results) => {
-    res.send('success');
+    // res.redirect('/login')
+    passport.authenticate('local', (err, user, info) => {
+      if(err) {
+        console.log('11')
+        return next(err)
+      }
+      if(!user) {
+        console.log('22')
+        res.status(401).send('Incorrect Password');
+      }else {
+        console.log('33')
+        const {first_name, last_name, email, id, balance} = user;
+        const obj = {}
+        obj.first_name = first_name;
+        obj.last_name = last_name;
+        obj.email = email;
+        obj.id = id;
+        obj.balance = balance
```

Comment:
There's an opportunity here to use a ES6 shorthand for defining the property name and values in an object just using a variable. 
`const obj = {first_name, last_name, email, id, balance };`
So, the property name with be the name of the variable name and the value will be the value of the variable.